### PR TITLE
Remove ads.txt sync functionality; for requests to /ads.txt issue a r…

### PR DESCRIPTION
**Pull request by omonbek22**
…edirect to Site GUID's specific ads.txt

### CODE REVIEW

File: Organic/Organic.php [hunk 1]
```diff
@@ -287,12 +287,10 @@ public function init( $apiUrl = null, $cdnUrl = null ) {
         /* Load up our sub-page configs */
         new AdminSettings( $this );
         new CCPAPage( $this );
-        new AdsTxt( $this );
```
Comment (sergunich) [inline Organic/Organic.php:290]: Have you tested these changes?
The `AdsTxt` ads custom action to  WP `init` filter during the initialization:
https://github.com/orgnc/wordpress-plugin/blob/d878e613952990f212f426124be6e3a7a9369e68/Organic/AdsTxt.php#L22

So.. without initialization of `AdsTxt` all of this won't work I suppose

File: Organic/AdsTxt.php [hunk 2]
```diff
@@ -22,24 +23,18 @@ public function __construct( Organic $organic ) {
     }
 
     public function get() {
```
Comment (sergunich) [inline Organic/AdsTxt.php:25]: I think we should rename the method considering that it just returns URL where to redirect

File: Organic/AdsTxt.php [hunk 3]
```diff
@@ -22,24 +23,18 @@ public function __construct( Organic $organic ) {
...
     public function show() {
         if ( isset( $_SERVER ) && $_SERVER['REQUEST_URI'] === '/ads.txt' ) {
             $enabled = $this->organic->getOption( 'organic::enabled' );
 
             if ( $enabled ) {
-                $adsTxt = $this->organic->getOption( 'organic::ads_txt' );
-                header( 'content-type: text/plain; charset=UTF-8' );
-                header( 'cache-control: public, max-age=86400' );
-                echo $adsTxt;
+                header( 'Location: ' . $this->get() );
```
Comment (sergunich) [inline Organic/AdsTxt.php:34]: Oh! And probably we should add a comment that only one redirect is allowed for `/ads.txt`
Per Ads.txt specification:
> Only a single HTTP redirect to a destination outside the original root domain is allowed to facilitate one-hop delegation of authority to a third party's web server domain.

It's not an issue in this case but comment could help prevent problems in future when someone else will work on this

File: Organic/AdsTxt.php (hunk 2 above)
Comment (omonbek22) [reply Organic/AdsTxt.php]: Agreed - will rename.

File: Organic/AdsTxt.php (hunk 3 above)
Comment (omonbek22) [reply Organic/AdsTxt.php]: I'll add a comment.

File: Organic/Organic.php (hunk 1 above)
Comment (sergunich) [reply Organic/Organic.php:290]: Oh! I see, yeah. Didn't saw that 👍

File: Organic/AdsTxt.php [hunk 4]
```diff
@@ -21,25 +22,26 @@ public function __construct( Organic $organic ) {
         add_action( 'init', array( $this, 'show' ) );
     }
 
-    public function get() {
-         return $this->organic->getOption( 'organic::ads_txt' );
+    public function getAdsTxtUrl() {
```
Comment (sergunich) [inline Organic/AdsTxt.php:25]: Looks like there is one more place where `ads.txt` was used
https://github.com/orgnc/wordpress-plugin/blob/feature%2FEP-822%2Fplugin-ads-redirect/Organic/GraphQL.php#L144

This GQL API is used by `organic-web`
https://github.com/orgnc/organic-web/blob/master/src/lib/wordpress/api/fragments/organicConfigFragment.ts

Looks like this field is actually not used there, but you should check with Content Team to be sure and ask them about how they handle the `ads.txt`

**Review by glebofff — APPROVED** [review]
Overall, looks good and - i checked - it works. Good job. :-)

File: Organic/AdminSettings.php [hunk 5]
```diff
@@ -320,7 +343,19 @@ public function showSettings() {
                 <input type="hidden" name="organic_sync_ads_txt" id="organic_sync_ads_txt" value="true" />
                 <p><input type="submit" value="Sync ads.txt"/></p>
             </form>
-
+            </div>
+            <script>
+                var hideShowSyncAdsTxt = function() {
```
[the remark this answers is not in the document]
Comment (omonbek22) [reply Organic/AdminSettings.php]: updated

File: Organic/AdsTxtSyncCommand.php [hunk 6]
```diff
@@ -12,6 +12,10 @@ class AdsTxtSyncCommand {
 
     public function __construct( Organic $organic ) {
         $this->organic = $organic;
+        $ads_txt_redirect = $this->organic->getOption( 'organic::ads_txt_redirect_enabled' );
```
[the remark this answers is not in the document]
Comment (omonbek22) [reply Organic/AdsTxtSyncCommand.php]: updated

File: Organic/AdminSettings.php [hunk 7]
```diff
@@ -94,7 +94,7 @@ protected function syncSettings() {
         if ( $_SERVER['REQUEST_METHOD'] != 'POST' ) {
             return;
         }
-        if ( ! isset($_POST['organic_update'] ) ) {
+        if ( ! isset( $_POST['organic_update'] ) ) {
```
[the remark this answers is not in the document]
Comment (omonbek22) [reply Organic/AdminSettings.php:97]: squashed

**Pull request merged**